pkg/scheduler/internal/heap/heap.go

```diff
@@ -56,7 +57,7 @@ type heapData struct {
 	// should be deterministic.
 	keyFunc KeyFunc
 	// lessFunc is used to compare two objects in the heap.
-	lessFunc LessFunc
+	lessFunc podinfo.LessFunc
```

Comment:
It feels to me like `LessFunc` belongs in this `heap` package. Is it still used anywhere within the `podinfo` package?

Reply:
It looks like `LessFunc` is used by both `SortableList` and the heap package. If I had a lot of time on my hands, there are ways I would want to reorganize/refactor this. But I think it's good enough as-is.

Reply:
I agree, perhaps a TODO within the codebase?